goss.yaml

```diff
@@ -12,3 +12,10 @@ command:
+  /test_scripts/test_execlineb_binary.sh:
```

Comment:
Does this need to be a script? I think you can test it directly in goss using the file test in goss https://github.com/aelsabbahy/goss/blob/master/docs/manual.md#file

Reply:
Updated to file test.